Detectors/EMCAL/workflow/include/EMCALWorkflow/RecoWorkflow.h

```diff
@@ -28,20 +28,22 @@ namespace reco_workflow
 /// \enum InputType
 /// \brief Input types of the workflow
 /// \ingroup EMCALworkflow
-enum struct InputType { Digits,  ///< read digits from file
-                        Cells,   ///< read compressed cells from file
-                        Raw,     ///< read data in raw page format from file
-                        Clusters ///< read native clusters from file
+enum struct InputType { Digits,    ///< read digits from file
+                        Cells,     ///< read compressed cells from file
+                        Raw,       ///< read data in raw page format from file
+                        Clusters,  ///< read native clusters from file
+                        BoostHisto ///<read in boost histograms from file
 };
 
 /// \enum OutputType
 /// \brief Output types of the workflow
 /// \ingroup EMCALworkflow
-enum struct OutputType { Digits,          ///< EMCAL digits
-                         Cells,           ///< EMCAL cells
-                         Raw,             ///< EMCAL raw data
-                         Clusters,        ///< EMCAL clusters
-                         AnalysisClusters ///< EMCAL analysis clusters
+enum struct OutputType { Digits,           ///< EMCAL digits
+                         Cells,            ///< EMCAL cells
+                         Raw,              ///< EMCAL raw data
+                         Clusters,         ///< EMCAL clusters
+                         AnalysisClusters, ///< EMCAL analysis clusters
+                         BadChannelMap     ///< EMCAL bad channel map
```

Comment:
Not needed in reco workflow, the calibration params are handled via the internal CCDB interface. I guess you mixed up something with the calibrator workflow.